content-scripts/cs.js

```diff
@@ -660,17 +660,19 @@ const showBanner = () => {
             .outerHTML,
           */
           Object.assign(document.createElement("a"), {
-            href: "https://scratch.mit.edu/scratch-addons-extension/settings?source=updatenotif",
+            // href: "https://scratch.mit.edu/scratch-addons-extension/settings?source=updatenotif",
+            href: "https://scratchaddons.com/feedback?utm_source=extension&utm_medium=updatenotification&utm_campaign=mv3",
             target: "_blank",
             style: NOTIF_LINK_STYLE,
-            textContent: chrome.i18n.getMessage("scratchAddonsSettings"),
+            // textContent: chrome.i18n.getMessage("scratchAddonsSettings"),
+            textContent: chrome.i18n.getMessage("feedback"),
```

Comment:
Could this be a new string? In some languages, it might be necessary to translate the link differently when it's part of a sentence.

Reply:
Sure